vnext/ReactUWP/Views/RawTextViewManager.cpp

```diff
@@ -53,12 +53,15 @@ void RawTextViewManager::UpdateProperties(
     if (propertyName == "text") {
       run.Text(asHstring(propertyValue));
       if (nodeToUpdate->GetParent() != -1) {
-        auto parent = this->m_wkReactInstance.lock()
-                          ->NativeUIManager()
-                          ->getHost()
-                          ->FindShadowNodeForTag(nodeToUpdate->GetParent());
-        if (parent->m_children.size() == 1) {
-          auto view = static_cast<ShadowNodeBase &>(*parent).GetView();
+        auto instance = this->m_wkReactInstance.lock();
+        const ShadowNodeBase *parent = nullptr;
+        if (instance != nullptr) {
```

Comment:
>!= nullptr [](start = 21, length = 10)

nit: remove != nullptr
see ES.87: Don't add redundant == or != to conditions in [https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md](https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md)

Reply:
this is a change in the convention used so far in the codebase, we have 170+ existing cases just in the uwp specific files to cleanup

Reply:
fixed these